### DISCUSSION TITLE
Files deleted could be marked clearer

### DISCUSSION THREAD

**Discussion by freiron** [community/community#76283] [Mobile]
For both light and dark mode I think there should be some red to signal that the file has been deleted. Because for binary files you only get the "binary files are not rendered" when it has been deleted.

(Middle file is removed)
![Screenshot_20231120_115232_GitHub.jpg](https://github.com/community/community/assets/111575175/9a6c7d44-a1d2-4bd1-890d-7aa325ab2389)

**Comment by Smalsss** [comment]
> For both light and dark mode I think there should be some red to signal that the file has been deleted. Because for binary files you only get the "binary files are not rendered" when it has been deleted.
> 
> (Middle file is removed)
> ![Screenshot_20231120_115232_GitHub.jpg](https://github.com/community/community/assets/111575175/9a6c7d44-a1d2-4bd1-890d-7aa325ab2389)
> 
>
